clients/go/gengo.go

```diff
@@ -99,32 +94,19 @@ func New(basePath string, logger wcl.WagClientLogger, transport *http.RoundTripp
 	basePath = strings.TrimSuffix(basePath, "/")
 	base := baseDoer{}
 	
-	// For the short-term don't use the default retry policy since its 5 retries can 5X
-	// the traffic. Once we've enabled circuit breakers by default we can turn it on.
+	// Don't use the default retry policy since its 5 retries can 5X the traffic
 	retry := retryDoer{d: base, retryPolicy: SingleRetryPolicy{}}
 
-	circuit := &circuitBreakerDoer{
-		d:     &retry,
-		// TODO: INFRANG-4404 allow passing circuitBreakerOptions
-		debug: true,
-		// one circuit for each service + url pair
-		circuitName: fmt.Sprintf("{{.ServiceName}}-%%s", shortHash(basePath)),
-		logger: logger,
-	}
-	circuit.init()
-
 	client := &WagClient{
 		basePath: basePath,
-		requestDoer: circuit,
+		requestDoer: &base,
```

Comment:
Should this be `retryDoer{d: base, retryPolicy: SingleRetryPolicy{}}`? We want to retry at least 1 time.

Reply:
Nice catch, annoyingly I did this right the first time around and missed it this time 😞